### PR TITLE
Pktdesk.com_PKT_technical_content_in_user_friendly_language

### PR DESCRIPTION
**Pull request by aamirmursleen**
PKT Desk aims to teach people about PKT from mining to starting a pool in a way that would make sense to a 10-year-old child.

### CODE REVIEW

**Comment by qskwood** [comment]
Did you intend to delete the template?

**Comment by aamirmursleen** [comment]
> Did you intend to delete the template?

Nope. I wanted to change the title.

**Comment by cjdelisle** [comment]
Hello @aamirmursleen , this is not an opinion of the NS but rather just my personal comments.
A project which helps inform the community and welcome new-comers to the project is clearly useful, but there are certain risks which need to be mitigated. Firstly, a project which communicates with the public carries a high risk of potential unethical communication, and therefore specific attention must be paid to proving that the communication will be ethical in nature. As it stands there is no mention of this at the moment, so on this point I would see it unlikely that this project would be funded.

A second issue is the question of what is being delivered to the community, when you do SEO for your website that benefits your domain which is owned by you. There isn't really any way for you to transfer ownership of your website to "the community" in the way that you can transfer code or documentation by licensing it as open source. And projects which benefit the project holder more than anyone else in the community will fall afoul of the [unfairly benefitting](https://github.com/pkt-cash/ns-projects/blob/master/acceptance_process.md#5-hazard-control) clause of the NS's guidelines.

To sum it up, I really like what you're doing and I want to find a way to support it, but I think we should work together to find a model for this which is not based on the Network Steward funding system because these two complexities above make it very difficult to fit into this into the NS funding structure.

However, this is just my personal opinion and if you do nothing at all, this will still be evaluated on the merits by the NS team, if you feel like there are changes which you can make that will make this a stronger proposal, you can make them until midnight UTC at the end of March 4.

**Comment by aamirmursleen** [comment]
Hi @cjdelisle [cjdelisle](https://github.com/cjdelisle). 

Open Source in Nature
-
All of the code is licensed under the Apache License 2.0 which outlines what users are allowed to do with the licensed code. Under this license, users can: Use the code commercially.

All Textual content, animations, drawings, logos, and design concepts are licensed under Creative Commons Attribution 4.0 where attribution of "PKTDesk" is considered valid attribution.

Users will be able to submit changes to the blog through Github, and administrators will review them according to the ethical communication guidelines.

Communication
-
The website carries a high risk of potential unethical communication, and therefore specific attention must be paid to proving that the communication will be ethical in nature. 

To ensure that the website complies with the guidelines, it will undergo strict approval standards. The existing content, as well as future content, will be adapted to the guidelines. 

Benefits to the Community
-
Throughout the website, we have attempted to provide answers to questions posed by members of the community. Using this website, they have been able to address questions they could not find elsewhere on the Internet, including how to buy PKT, convert to WPKT, and mine on Android. 

Since most people do not know how to buy PKTs and WPKTs or other technical work like mining, the content is extremely important. When users possess this knowledge, the project is likely to grow.

We will add more blogs over time as the need arises. The community members will be able to submit their blogs related to PKT on PKTDesk that will be evaluated based on the guidelines. 

According to your statement, since the project is owned by me, I can give ownership of the domain and hosting to any person to ensure best practices and standards are followed.

**Comment by cjdelisle** [comment]
Congratulations, your project has been selected for funding.

As a point of order, the version of the project which has been selected is the last version that was submitted before the deadline: https://github.com/pkt-cash/ns-projects/blob/979c3532ca39e9c051e76dd94f0102869675c1a9/projects/2022_3_3_pktdesk_blog.md

I invite you to examine the comments left by the reviewers which are found in the vote file here: https://github.com/pkt-cash/ns-projects/blob/master/vote_2022_02_10.py and I wish you an excellent project ahead.

Caleb
